content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,24 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-rc.2 [2020-10-22]
+
+### Features
+
+- Upgrade to [Flux v0.90.0](/influxdb/v2.0/reference/release-notes/flux/#v0-90-0-2020-10-19).
+- Add `--force` option to the `influx stacks rm` command, which lets you remove a stack without the confirmation prompt.
+- Add `aggregate_resultset` for mean aggregate pushdown to optimize window results.
+- Return an error if adding a resource to a stack (`influx stacks update --addResource`) fails due to an invalid resource type or resource ID.
+
+### Bug Fixes
+
+- Update `pkger` test templates to use valid Flux to avoid `found unexpected argument end` error. Previously, any template with a `v.dashboardVariable` returned an `undefined identifier v` error.
+- Update the InfluxDB configuration file `/etc/influxdb/influxdb.conf` to recognize the user's home directory. Previously, if a user (other than root user) ran the `upgrade` command, a permissions error occurred.
+- Remove the Telegraf RAS Daemon plugin and other miscellaneous Telegraf plugin updates.
```

Comment:
do we want to say anything specific about misc plugin updates?

Reply:
they were mostly just minor text changes to the readme's so i think we are ok